plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -34,7 +75,17 @@ export const endSpan = (
 export const defaultDbStatementSerializer: DbStatementSerializer = (
   cmdName,
   cmdArgs
-) =>
-  Array.isArray(cmdArgs) && cmdArgs.length
-    ? `${cmdName} ${cmdArgs.join(' ')}`
-    : cmdName;
+) => {
+  if (Array.isArray(cmdArgs) && cmdArgs.length) {
+    const argsSubset = SerializationSubsets[cmdName.toUpperCase()];
+    if (argsSubset) {
+      const args = cmdArgs.slice(0, argsSubset);
+      const remainder = cmdArgs.slice(argsSubset);
+      return `${cmdName} ${args.join(' ')} [${
+        remainder.length
```

Comment:
Should we write "[0 other arguments]" when `remainder.length` is zero?

below, when we serialize all arguments, we write just `${cmdName} ${cmdArgs.join(' ')}` without the "other arguments" part so it make sense to me to drop it here as well for consistency.

Reply:
Can do.